src/core/interactive-creative-file.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Identify the file and the framework needed for execution of a media file.
+ *
+ * @class InteractiveCreativeFile
+ * @since VAST 4.0
+ * @author Tim De Pauw <tim.depauw@zentrick.com>
+ * @copyright © 2018 Zentrick nv
+ */
+export class InteractiveCreativeFile {
+  // Attribute(s).
+
+  /**
+   * The MIME type of the resource being served.
+   *
+   * @type {?string}
+   */
+  get type () {
+    return this._type
+  }
+
+  set type (value) {
+    this._type = value
+  }
+
+  /**
+   * The API needed to execute the Icon resource file if applicable.
+   *
+   * @type {?string}
+   */
+  get apiFramework () {
+    return this._apiFramework
+  }
+
+  set apiFramework (value) {
+    this._apiFramework = value
+  }
+
+  // Content.
+
+  /**
+   * A CDATA-wrapped URI to a file providing creative functions for the media
```

Comment:
CDATA is an XML feature which is only relevant for the parser, so it's not relevant here.